set.mm

```diff
@@ -526906,6 +526911,67 @@ Real and complex numbers (cont.)
       ASACUETBDFTUB $.
   $}
 
+  ${
+    $d A x $.
+    $( The union of a finite ordinal is a finite ordinal.  (Contributed by
+       Scott Fenton, 17-Oct-2024.) $)
+    nnuni $p |- ( A e. _om -> U. A e. _om ) $=
+      ( vx com wcel c0 wceq cv csuc wrex cuni nn0suc unieq uni0 eqtrdi eqeltrdi
+      wo peano1 word nnord syl ordunisuc id eqeltrd eleq1d syl5ibrcom rexlimiv
+      jaoi ) ACDAEFZABGZHZFZBCIZPAJZCDZBAKUHUNULUHUMECUHUMEJEAELMNQOUKUNBCUICDZ
+      UNUKUJJZCDUOUPUICUOUIRUPUIFUISUIUATUOUBUCUKUMUPCAUJLUDUEUFUGT $.
+  $}
+
+  ${
+    $d A x y $.  $d B x y $.
+    $( Uniqueness of finite ordinal subtraction.  (Contributed by Scott Fenton,
```

Comment:
Since there is no subtraction appearing in the statement, another phrasing can be:

Finite ordinal subtraction is left-cancellative.

Reply:
Done